arangod/Cluster/ClusterMethods.cpp

```diff
@@ -2570,55 +2542,67 @@ std::vector<std::shared_ptr<LogicalCollection>> ClusterMethods::persistCollectio
 
 std::string const apiStr("/_admin/backup/");
 
-arangodb::Result hotBackupList(std::vector<ServerID> const& dbServers, VPackSlice const payload,
+arangodb::Result hotBackupList(network::ConnectionPool* pool,
+                               std::vector<ServerID> const& dbServers, VPackSlice const payload,
                                std::unordered_map<std::string, BackupMeta>& hotBackups,
                                VPackBuilder& plan) {
+  TRI_ASSERT(pool);
   hotBackups.clear();
 
   std::map<std::string, std::vector<BackupMeta>> dbsBackups;
 
-  auto cc = ClusterComm::instance();
-  if (cc == nullptr) {
-    // shutdown, leave here
-    return TRI_ERROR_SHUTTING_DOWN;
-  }
-
-  auto body = std::make_shared<std::string>(payload.toJson());
+  VPackBufferUInt8 body;
+  body.append(payload.begin(), payload.byteSize());
+  network::RequestOptions reqOpts;
+  reqOpts.retryNotFound = false;
+  reqOpts.skipScheduler = true;
+  
   std::string const url = apiStr + "list";
 
-  std::vector<ClusterCommRequest> requests;
+  std::vector<Future<network::Response>> futures;
+  futures.reserve(dbServers.size());
   for (auto const& dbServer : dbServers) {
-    requests.emplace_back("server:" + dbServer, RequestType::POST, url, body);
+    futures.emplace_back(network::sendRequest(pool, "server:" + dbServer,
+                                              fuerte::RestVerb::Post, url, body, reqOpts));
   }
-
-  // Perform the requests
-  auto nrGood = cc->performRequests(
-    requests, CL_DEFAULT_TIMEOUT, Logger::BACKUP, false, false);
-
-  LOG_TOPIC("410a1", DEBUG, Logger::BACKUP) << "Got " << nrGood << " of " << requests.size() << " lists of local backups";
+  
+  size_t nrGood = 0;
+  for (Future<network::Response>& f : futures) {
+    network::Response const& r = f.get();
+    if (r.ok()) {
```

Comment:
I guess this should be
```suggestion
    if (!r.ok()) {
```